boards.go

```diff
@@ -29,6 +29,24 @@ type Board struct {
 	ModifiedAt        *string            `json:"modified_at,omitempty"`
 }
 
+// BoardLite represents a simplify dashboard (without widgets, notify list, ...)
+// It's used when we load all boards.
+type BoardLite struct {
+	Title        *string `json:"title"`
+	Description  *string `json:"description,omitempty"`
+	LayoutType   *string `json:layout_type`
```

Comment:
You forgot quotes around `layout_type` (plus also make it `omitempty`, please).

Reply:
amended